environment.yml

```diff
@@ -0,0 +1,20 @@
+name: symbiflow_arch_def_base
```

Comment:
Floating version is likely not going to suitable, given how sensitive we currently are to the version of the yosys and vtr.

Reply:
As you said, I don't think anyone would use the floating version -- the "floating" version is more used to generate the pinned version (frequently by an automatic robot?).

Reply:
> As you said, I don't think anyone would use the floating version -- the "floating" version is more used to generate the pinned version (frequently by an automatic robot?).

For better or worse, we don't have a way to do this yet.  The "pinned" version is also overly specific, as it ends up specifying every package, rather than the ones we care about.  Is that what you mean?